ci/tools/run-tests

```diff
@@ -9,14 +9,31 @@
 set -euo pipefail
 
 # Check if the script was called with exactly 1 argument
-if [[ ${#} -ne 1 && ( "${1}" == "bindings" || "${1}" == "core" ) ]]; then
-  echo "Error: This script requires exactly 1 argument (what is tested). You provided ${#}"
-  echo "Usage: ${0} test_module[bindings or core]"
+if [[ ${#} -ne 1 ]]; then
+  echo "Error: This script requires exactly 1 argument. You provided ${#}"
+  exit 1
+fi
+if [[ "${1}" != "bindings" && "${1}" != "core" && "${1}" != "pathfinder" ]]; then
+  echo "Error: Invalid test module '${1}'. Must be 'bindings', 'core', or 'pathfinder'"
   exit 1
 fi
 
 test_module=${1}
 
+# Unconditionally install pathfinder wheel
+# (it is a direct dependency of bindings, and a transitive dependency of core)
+pushd ./cuda_pathfinder
+echo "Installing pathfinder wheel"
+pwd
+ls
+pip install $(ls *.whl)[test]
+if [[ "${test_module}" == "pathfinder" ]]; then
+  echo "Running pathfinder tests"
+  pwd
+  pytest -ra -s -v tests/
+fi
```

Comment:
We need to either move pathfinder tests to after cuda-core tests, or have a separate pipeline dedicated to it. Right now, we rely on cuda-bindings/cuda-core installing the dependencies for us. If we run this test first, we don't have access to any CTK components in the wheel pipelines (local CTK ones are fine).

Reply:
Done: commit fd18e1e2a7dbb4d102c123c300d476405425aaf2